CHANGELOG.rst

```diff
@@ -22,6 +22,13 @@ individual files.
 
 The changes are now listed with the most recent at the top.
 
+**September 20 2022 :: Feature request for ROMS model_mod for Tag: v10.3.3**
```

Comment:
This is a style suggestion, since it is a new feature I would just state what it is, e.g.

```suggestion
**September 21 2022 :: ROMS model_mod perturbation routine Tag: v10.4.0**
```